lib/routes/v1.js

```diff
@@ -169,6 +183,18 @@ api.declare({
 
   let modDate = new Date();
 
+  // We want to make sure that all AMIs that we are submitting are valid
+  for (let region of input.regions) {
+    let validAmi = await amiExists(this.ec2, region.launchSpec.ImageId, region.region);
+    if (!validAmi) {
```

Comment:
I'm not aware of the behavior, but what happens when the underlying code throws an error? does our api handling return back a certain status code then with the error?  Do we want to catch the error and then include the ami and region that was used in the response?